.github/scripts/update.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Increment position (Patch is default)
+position=2
+
+# set nocasematch option
+shopt -s nocasematch
+
+if [[ "$1" =~ "^major$" ]]; then
+  position=0
+elif [[ "$1" =~ "^minor$" ]]; then
+  position=1
+fi
+
+# unset nocasematch option
+shopt -u nocasematch
+
+# Increment version: usage :: sh increment_version <VERSION> 0|1|2
```

Comment:
Should we be more explicit or add a call example? Without looking at the calling script, it's a bit hard for me to understand the use

Reply:
ok, will do it